README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About Me
+
+Hello! My name is **Nikhil Mandalaparthy** (my first name is pronounced *nick-hill*). I'm currently a MPP student at the Harris School of Public Policy, and I just graduated in June from the College, with a BA in Public Policy Studies. This is my [personal website](https://nikhilmandala.com/).
```

Comment:
Good use of bolded and italicized text. The link is properly embedded in the text.

Reply:
Bolded and italicized have been properly conducted in the text.